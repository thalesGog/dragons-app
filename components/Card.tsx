import styled from 'styled-components'

const Card = styled.div`
  display: flex;
  align-items: center;
  flex-direction: column;
  border-radius: 0.125rem;
  background: #fff;
  margin-top: 1.5rem;
  padding: 1rem 1.5rem;
  font-size: 1.25rem;
  text-align: start;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
  box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
  p,
  h3 {
    margin: 8px;
  }
`

export default Card
