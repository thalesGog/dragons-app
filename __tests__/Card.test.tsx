import React from 'react'
import {render} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import 'jest-styled-components'
import Card from '../components/Card'

test('Render Card component', async () => {
  const {container} = render(<Card />)
  expect(container).toMatchSnapshot()
})
