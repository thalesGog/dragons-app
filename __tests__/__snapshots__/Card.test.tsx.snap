// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Card component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  border-radius: 0.125rem;
  background: #fff;
  margin-top: 1.5rem;
  padding: 1rem 1.5rem;
  font-size: 1.25rem;
  text-align: start;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
  border-bottom: 1px solid #10162f;
}

.c0 p,
.c0 h3 {
  margin: 8px;
}

<div>
  <div
    class="c0"
  />
</div>
`;
